honeybee_radiance_command/options/rmtxop.py

```diff
@@ -0,0 +1,37 @@
+"""Rmtxop parameters"""
+from .optionbase import OptionCollection, BoolOption, StringOptionJoined
+
+
+class RmtxopOptions(OptionCollection):
+    """
+    [-v] [-f[afdc]]
+
+    Also see: https://www.radiance-online.org/learning/documentation/manual-pages/pdfs/rmtxop.pdf
+    """
+
+    __slots__ = ('_v', '_f',)
+
+    def __init__(self):
+        """rmtxop command options."""
+        OptionCollection.__init__(self)
+        self._on_setattr_check = False
+        self._v = BoolOption('v', 'Turn on verbose reporting. Defaults to False')
+        self._f = StringOptionJoined('f', 'output data format. Set "a" for ASCII, "d" for'
+                                          ' binary doubles, "f" floats and "c" for RGBE'
+                                          'colors', valid_values=('a', 'f', 'd', 'c'))
+
+    @property
+    def v(self):
+        return self._v
+
+    @v.setter
+    def v(self, value):
+        self._v = value
```

Comment:
Shouldn't this be `self._v.value = value`?

```suggestion
        self._v.value = value
```

Reply:
@mostaphaRoudsari Yup, these are errors. I need to think of a way to ensure that these can checked automatically once and for all. Because it seems like a recurring PITA. I fixed similar bugs in evalglare and dcglare but now am repeating those myself :|